declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc

```diff
@@ -0,0 +1,63 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="customizing-permissions-by-creating-aggregated-cluster-roles"]
+= Customizing permissions by creating aggregated cluster roles
+:context: customizing-permissions-by-creating-aggregated-cluster-roles
+
+toc::[]
+
+The default cluster role for the Argo CD Application Controller has a specific set of hard-coded permissions. The {gitops-title} Operator manages this cluster role, so you cannot modify it. As a cluster administrator, you can customize the permissions by using any one of the following methods:
+
+* xref:../declarative_clusterconfig/customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances.adoc#customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances[Creating user-defined cluster roles]
+* xref:../declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc#gitops-creating-aggregated-cluster-roles_customizing-permissions-by-creating-aggregated-cluster-roles[Creating aggregated cluster roles]
+
+With aggregated cluster roles, you can aggregate several cluster roles into a single combined cluster role. You can enable users to easily add user-defined permissions for the Argo CD Application Controller. You do not have to define permissions by creating a completely new cluster role from scratch.
+
+[IMPORTANT]
+====
+* You can create aggregated cluster roles only for the Argo CD Application Controller component of a cluster-scoped Argo CD instance.
+* Deleting the `aggregatedClusterRoles` field from the Argo CD custom resource (CR) does not delete the user-defined cluster role. You have to manually delete the user-defined cluster role using the CLI or UI for the efficient use of resources and memory.
+====
+
+[id="prerequisites_{context}"]
```

Comment:
Prereqs should really be specific to each procedure, they should not be in the assembly (or repeat needed items in each procedure). If I go to do something, but don't look here, I'll miss important information.